site/content/en/docs/tasks/run_flux.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Run A Flux Minicluster"
+date: 2022-02-14
+weight: 6
+description: >
+  Run a Kueue scheduled Flux minicluster.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [Flux Operator's](https://flux-framework.org/flux-operator/) Mini Clusters.
+
+For more information on Flux Operator visit [Flux Operator](https://flux-framework.org/flux-operator/).
+
+
+## Before you begin
+
+Please check [Run A Job](/docs/tasks/run_jobs) for details on the initial cluster setup and queue configuration.
+
+## 1. MiniCluster definition
+
+Since a Flux MiniCluster uses  a `batch/job` to run the actual workload, no extra components are needed for it to be managed by Kueue, However the following aspects should be taken into account:
+
+### a. Queue selection
+
+The target local queue should be specified in the `spec.jobLabels` section of the minicluster configuration.
+
+```yaml
+  jobLabels:
+    kueue.x-k8s.io/queue-name: user-queue
+```
+
+### b. Configure the resource needs
+
+The resource needs of the workload can be configured in the `spec.container[].resources` sections of the minicluster configuration.
+
+```yaml
+spec:
+  containers:
+    - image: <image>
+      resources:
+        requests:
+          cpu: 4
+          memory: "200Mi"
+```
+
+## 2. Configuration example
+
+```yaml
+apiVersion: flux-framework.org/v1alpha1
+kind: MiniCluster
+metadata:
+  generateName: flux-sample-kueue-
+spec:
+  size: 1
+  containers:
+    - image: ghcr.io/flux-framework/flux-restful-api:latest
+      command: echo hello world 
+      resources:
+        requests:
+          cpu: 4
+          memory: "200Mi"
+  jobLabels:
+    kueue.x-k8s.io/queue-name: user-queue
+```
+
+## 3. Known Limitations
+
+- The certificate generation step of the MiniCluster life cycle runs outside it's main job and it's execution will not be managed by Kueue.
+- In some configuration variants the MiniCluster's job might not run to completion blocking the resources of the associated queue until it's deletion.
```

Comment:
```suggestion
- In some configuration variants, the MiniCluster's job might not run to completion blocking the resources of the associated queue until it's deletion.
```

I don't think this will change anytime soon, or is it @vsoch?

Reply:
I'm definitely up to try out different strategies - what we really need is to have the logic afforded by the new JobSet. My only idea was to go off of what you mentioned about parent/child relationships @alculquicondor  and make the broker pod a separate job that is launched first, and then have the other cluster workers somehow children? (is there an example of this somewhere I haven't done it before!) But that might be more effort / refactor than it's worth if we eventually want to go back to one IndexedJob with more structure.

Reply:
we could leave this in and remove it later on

Reply:
removed